jupyter_notebook/README.md

```diff
@@ -0,0 +1,81 @@
+# Jupyter Notebook
+
+This directory contains the Jupyter notebook support for
+the FlexFlow Python binding.
+
+## Quick Start
+### Pre-requisite
+* Python >= 3.6
+* FlexFlow Python binding needs to be installed, please check the [installtion guide](https://github.com/flexflow/FlexFlow/blob/master/INSTALL.md)
+* Install Jupyter notebook
+
+        pip install notebook
+
+### Install the Legion IPython kernel
+```
+python ./install.py --(configurations)
+```
+Please refer to the [Kernel Configurations](#kernel-configurations) section for the configuration details.
+
+If the installtion is successed, the following log will be printed to the terminal.
+The `flexflow_kernel_nocr` is the kernel name, and the `FlexFlow_SM_GPU` is the display name
+of the kernel, which can be modified by the configuration json file. 
+`FlexFlow` is the name entry in the json file, `SM` means the kernel
+is only for shared memory machine, and `GPU` meams GPU execution is enabled. 
+```
+IPython kernel: flexflow_kernel_nocr(FlexFlow_SM_GPU) has been installed
+```
+The installed kernel can be also seen by using the following command:
+```
+jupyter kernelspec list
+```
+
+### Start the Jupyter Notebook server
+* Launch jupyter notebook server on the computing node
+
+        jupyter notebook --port=8888 --no-browser
+
+
+* Create a turnel between localhost to the computing node, this step is not needed if
+the jupyter server is run on the local machine.
+
+        ssh -4 -t -L 8888:localhost:8002 username@login-node-hostname ssh -t -L 8002:localhost:8888 computing_node
+
+### Use the Jupyter Notebook in the browser
+* Open the browser, type the addredd http://localhost:8888/?token=xxx, the token will be
+displayed in the terminal once the server is started. 
+* Once the webpage is loaded, click "New" on the right top corner, and click the kernel 
+just installed. It is shown as the display name of the kernel, e.g. `FlexFlow_SM_GPU`.
+
+### Uninstall the kernel
```

Comment:
It might be clearer to replace "kernel" everywhere with "jupyter kernel". I was confused for a while thinking "kernel" referred to the operator kernels